### ISSUE TITLE
`load_tooluse_dataset` returning tuple causes error running

### ISSUE THREAD

**Issue by tylerkastner** [idanshen/Self-Distillation#7]
Currently `load_tooluse_dataset` returns a tuple:
https://github.com/idanshen/Self-Distillation/blob/1825ef3074b6289fffe285145ce84522c1f059d3/main.py#L46

This causes an error when being passed to `DistilTrainer` later on which causes the repo to be unrunnable. This can be fixed only returning `train_dataset`, or by changing 
https://github.com/idanshen/Self-Distillation/blob/1825ef3074b6289fffe285145ce84522c1f059d3/main.py#L60
to `dataset, _ = load_tooluse_dataset(args.seed)`. Is there any reason for the second argument? I.e., which fix should be taken?
